input/http.js

```diff
@@ -11,7 +11,7 @@ function create ({name = 'HTTP server', path, parse}) {
         let messages = Array.isArray(req.body) ? req.body : [req.body]
         messages.forEach(message => {
           try {
-            const log = parse(req.body)
+            const log = parse(message)
```

Comment:
I would get rid of `format/json` that do not do anything anymore. Which makes sense since we do not need to do anything to translate our own format :)

And I would just parse the whole body and transform into an immutable map directly in the http input.

Reply:
This comment is more for the other side:
https://github.com/access-watch/access-watch/pull/39

Reply:
Syslog?

Reply:
Example: https://github.com/access-watch/access-watch-express-logger

```
const syslogInput = input.syslog.create({
  name: 'Syslog (JSON standard format)',
  port: 1516,
  parse: format.json.parser()
})

pipeline.registerInput(syslogInput)
```

Reply:
I don't think it makes sense to have this json format adapter anymore. What the inputs should do if they're not passed a format parser is assume it is a well-formed JSON object that is passed to them and just build an immutable map with `fromJS`.